caravel/assets/javascripts/SqlLab/components/TableElement.jsx

```diff
@@ -65,11 +67,8 @@ class TableElement extends React.PureComponent {
     this.setState({ sortColumns: !this.state.sortColumns });
   }
 
-  render() {
+  getHeader() {
```

Comment:
for any method that returns a node/element it's a nice pattern to name them with `render`, so  this would be `renderHeader`,